samples/gke_enable_stackdriver_monitoring.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPGKEEnableStackdriverMonitoringConstraintV1
+metadata:
+  name: enable_gke_stackdriver_monitoring
+  annotations:
+    benchmark: GKE_HARDENING_GUIDELINE
```

Comment:
This is part of CIS, can you change this to CIS11_7.02 and add another description field to describe the benchmark? e.g. https://github.com/forseti-security/policy-library/blob/master/samples/iam_blacklist_role.yaml